src/common/components/nav/nav.component.js

```diff
@@ -71,6 +75,13 @@ class NavController{
     this.changeMetaTag('viewport', this.menuType === 'mobile' ? 'width=device-width, minimum-scale=1.0' : 'width=1024');
   }
 
+  // eslint-disable-next-line no-unused-vars
+  giftAddedToCart(event, item) {
```

Comment:
There are there because angular events pass them along. I'm not sure how it would work leaving them off.